CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+1.13.0 (2020-05-14)
+
```

Comment:
You are missing the `-`. Also, usually we leave out the release date until we are ready to actually release, unless it is a given that we will release on that day.